yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ErrorDetails.java

```diff
@@ -39,24 +36,19 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
+
         ErrorDetails that = (ErrorDetails) o;
         return Objects.equals(code, that.code) && Objects.equals(description, that.description);
     }
 
     @Override
     public int hashCode() {
-        final int prime = 31;
-        int result = 1;
-
-        result = prime * result + ((code == null) ? 0 : code.hashCode());
-        result = prime * result + ((description == null) ? 0 : description.hashCode());
-
-        return result;
+        return Objects.hash(code, description);
     }
 
     @Override
     public String toString() {
-        return String.format("- Error[code='%s', description='%s']", code, description);
+        return String.format(" - Error[code='%s', description='%s']", code, description);
```

Comment:
(nit) Why does this have a ` - ` at the start of the string? I'd expect a `toString` to no have any "prefix" (callers can always prefix it in the way they see fit)

---

(nit) Also probably easier to survive refactorings if this was 
```java
%s[code='%s' ...]", this.getClass().getSimpleName(), code, ...)
```

Reply:
gonna change as suggested and handle spaces or `-` in the caller. I just managed to test and the final result is 

![image](https://user-images.githubusercontent.com/2480863/168050143-c6195bd1-4efd-45cc-b40b-3b7fb4054559.png)